src/molecules/TrysilPlug.js

```diff
@@ -87,20 +103,23 @@ const TrysilPlug = ({
 				{kicker && <Kicker order={1}>{kicker}</Kicker>}
 				{image
 				&& (
-					<LazyProgressiveImage
-						order={2}
-						alt={title}
-						ratio={ratio}
-						offset={offset}
-						src={placeholderUrl ? placeholderUrl : image}
-						preventBlur={preventBlur}
-						fadeIn={fadeIn}
-					>
-						{sources.length === 0
-						&& <Source srcSet={image} />
-						}
-						{sources.map((source, i) => <Source srcSet={source.url} media={source.media} key={`source-${i}`} />)}
-					</LazyProgressiveImage>
+					<Relative>
```

Comment:
We will not wrap our figure in a div.

Reply:
No clear, why not?
Ok, for such case I can use existing functionality, agree. But why not wrapping component in `div` for styling purposes?

Reply:
By the way, thanks, I haven't checked custom labels functionality. Works fine for me!